CONTRIBUTING.md

```diff
@@ -155,19 +155,47 @@ For Eclipse, go to `Preferences->Java->Installed JREs` and add `-ea` to
 
 ### Java Language Formatting Guidelines
 
+Java files in the Elasticsearch codebase are formatted with
+[google-java-format](https://github.com/google/google-java-format), which
+enforces the [Google Java
+Style](https://google.github.io/styleguide/javaguide.html). There are
+plugins available for Eclipse, JetBrains IDEs such as IntelliJ IDEA, and a
```

Comment:
Could we link directly to the plugins ? 
Alternatively @mark-vieira do you know a way to generate an IDEA configuration that would at least suggests users to install the plugin ?

Reply:
I'm not aware of any. I suspect something like installing a plugin is not something most folks want to happen when importing a project. There are likely security concerns there.